collect_app/src/main/res/layout/form_chooser_list_item_text.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:orientation="vertical"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1">
+
+    <!-- Material Design reference: https://material.io/design/components/lists.html#specs -->
+
+    <TextView
+            android:id="@+id/text1"
```

Comment:
These IDs have apparently been like this for [over 7 years](https://github.com/opendatakit/collect/blame/78281ed4389a96425c88f7bbe20df71e732a2f94/collect_app/src/main/res/layout/two_item.xml) -- I have no idea why, but I agree it's confusing.

They're used for different things in different views which is why I didn't touch them, but I will give them generic names at least. Perhaps `title`, `subtitleA`, `subtitleB`, etc.

The main point of confusion is that `text3` is **above** `text2` because it's an optional extra item that most views don't have, but the ones that do have it want it on top.